### ISSUE TITLE
Concept Insights - labelSearch() is always returning an empty array (i.e. [])

### ISSUE THREAD

**Issue by kajaldas007** [watson-developer-cloud/nodejs-wrapper#5]
params used: 
var params = {user: '5d05b04f-84d3-4875-a1b8-924a18886cf5',  // this is my user id
	  	       corpus: 'genome',
      	               query: 'miRNA'};

**Comment by germanattanasio** [comment]
@kajaldas007 I can't reproduce your error because you are using a private corpus which I can't access. 

I'm able to get results doing:

```js
var watson = require('watson-developer-cloud');

var conceptInsights = watson.concept_insights({
  username: 'INSERT YOUR USERNAME FOR THE CONCEPT INSIGHTS SERVICE HERE',
  password: 'INSERT YOUR PASSWORD FOR THE CONCEPT INSIGHTS SERVICE HERE',
  version: 'v1'
});

var payload = {
  corpus: 'ibmresearcher',
  user: 'public',
  func: 'labelSearch',
  limit: 4,
  prefix: true,
  concepts: true,
  query: 'DevOps'
};

conceptInsights.labelSearch(payload, function(error, results) {
  if (error)
    return console.log(error);
  else {
    return console.log(JSON.stringify(results, null, 2));
  }
});
```

Returns:

```js
[
  {
    "id": "DevOps",
    "label": "DevOps",
    "type": "concept",
    "result": {
      "id": "DevOps",
      "abstract": "DevOps (a portmanteau of development and operations) is a software development method that stresses communication, collaboration and integration between software developers and Information Technology(IT) professionals. DevOps is a response to the interdependence of software development and IT operations. It aims to help an organization rapidly produce software products and services.",
      "label": "DevOps",
      "type": "wiki_resource",
      "link": "http://en.wikipedia.org/wiki/DevOps"
    }
  }
]
```

**Comment by germanattanasio** [comment]
fixed in 7b446adc3226dbd99530f5f42e52c3dec949b8db
